policy/modules/system/systemd.te

```diff
@@ -277,6 +277,9 @@ init_system_domain(systemd_update_done_t, systemd_update_done_exec_t)
 type systemd_update_run_t;
 files_type(systemd_update_run_t)
 
+type systemd_user_home_t;
+userdom_user_home_content(systemd_user_home_t)
+
```

Comment:
this content is in ~/.config (xdg_conf_t) and ~/.local/share (xdg_data_t)
I would differentiate between the two (because afaik xdg_data_t is only used by systemd to store timer stamp files and maybe some other state but its AFAIK not used to store units in)

maybe:
xdg_conf_content(systemd_conf_home_t) ;; no need for a separate unit type for units in ~/.config/systemd/user
xdg_data_content(systemd_data_home_t)

Reply:
OK that makes sense.